test/Interop/Cxx/operators/Inputs/member-inline.h

```diff
@@ -477,4 +477,14 @@ struct HasOperatorCallWithDefaultArg {
   int operator()(int x = 0) const { return value + x; }
 };
 
+class HasStaticOperatorCallBase {
+public:
+    static int operator()(int x) {
+        return x + 42;
+    }
+};
+
+class HasStaticOperatorCallDerived: public HasStaticOperatorCallBase {
+};
+
```

Comment:
Could you please add a test where a struct declares both static and non-static `operator()`?

Reply:
Yep! that works in C++ when the function signatures are different, so I added a test case.